.github/workflows/CI.yml

```diff
@@ -15,41 +15,42 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - 1
+          - "1.0"  # LTS
```

Comment:
I feel like this has already been discussed but I've done so many of these CI changes that I don't remember. Adding the 1.0 here will add more than 1 1.0 test. Is this ok, or do we want just the single 1.0 test, in which I will revert these matrix changes.

Reply:
I usually just run 1 linux LTS job, but I don't think it matters too much. The tests should be pretty cheap.

Reply:
Seems fine to have multiple